packages/backend/src/analytics/LightdashAnalytics.ts

```diff
@@ -740,6 +741,32 @@ export type CreateSqlChartVersionEvent = BaseTrack & {
     };
 };
 
+export type CreateSemanticLayerChartVersionEvent = BaseTrack & {
+    event: 'semantic_layer_chart_version.created';
+    userId: string;
+    properties: {
+        chartId: string;
+        versionId: string;
+        projectId: string;
+        organizationId: string;
+        chartKind: ChartKind;
+        semanticLayerQuery: SemanticLayerQuery;
+        barChart?: {
```

Comment:
this does not have table config. do we also need to keep all configuration blobs as the analytics event or the active one @TuringLovesDeathMetal

Reply:
@TuringLovesDeathMetalsSorry for providing less than a zero context 😂 

I noticed that with SQL Runner, when you save a chart, we fire an analytics event. The analytics event includes all of the configurations for all charts, not just the active chart that's being used. Additionally, we are not including the table visualization configuration at all. I copied this "blindly" from SQL Runner but left a comment not to forget to update both places if needed.

Reply:
>The analytics event includes all of the configurations for all charts, not just the active chart that's being used.


wut - that sounds wild 😂 

Ya, can we _**just**_ include the active chart configuration. Also, if there's a table visualization, then include it's configuration